terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -0,0 +1,31 @@
+import graphene
+from graphene import relay
+from graphene_django import DjangoObjectType
+
+from apps.soilproj.models import Site
+
+from .commons import BaseWriteMutation, TerrasoConnection
+
+
+class SiteNode(DjangoObjectType):
+    id = graphene.ID(source="pk", required=True)
+
+    class Meta:
+        model = Site
+
+        filter_fields = {"name": ["icontains"]}
+        fields = ("name", "lat_deg", "lon_deg")
+
+        interfaces = (relay.Node,)
+        connection_class = TerrasoConnection
+
+
+class SiteAddMutation(BaseWriteMutation):
+    site = graphene.Field(SiteNode, required=True)
+
+    model_class = Site
+
+    class Input:
+        name = graphene.String(required=True)
+        lat_deg = graphene.Float(required=True)
+        lon_deg = graphene.Float(required=True)
```

Comment:
Suggestion: rename this to `latitude` (or `lat`) and `longitude` (or `lng`). You could also pick the long name and alias the short name as a function. We use `lat` an`lng` elsewhere in the code.

Reply:
+1

Reply:
I actually rely prefer having the units very explicitly included in the variable name. This helps avoid logic errors.

Reply:
This is nonstandard and will be confusing/resulting in errors. Other code (and other APIs) use lat/lng.

Reply:
This is nonstandard and will be confusing/resulting in errors. Other code (and other APIs) use lat/lng.